cardinal/world.go

```diff
@@ -347,14 +347,19 @@ func (w *World) StartGame() error {
 		}
 	}
 
-	// TODO(scott): hypothethically, this would be where we are calling RecoverFromChain()
-
 	// Recover pending transactions from redis
 	err := w.recoverAndExecutePendingTxs()
 	if err != nil {
 		return err
 	}
 
+	// If Cardinal is in Prod and Router is set, recover any old state of the engine from the chain
+	if getWorldConfig().CardinalMode == RunModeProd && w.router != nil {
+		if err := w.RecoverFromChain(context.Background()); err != nil {
+			return eris.Wrap(err, "failed to recover from chain")
+		}
+	}
```

Comment:
i think i'd prefer to put `mode` as a field on World rather than read the config again

Reply:
I agree, first thing I did was check world to see if it had a config! 🔥